modules/nf-core/gem2/gem2bedmappability/meta.yml

```diff
@@ -0,0 +1,55 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "gem2_gem2bedmappability"
+description: Convert a mappability file to bedgraph format
+keywords:
+  - mappability
+  - bedgraph
+  - index
+  - gem
+tools:
+  - "gem2":
+      description: "GEM2 is a high-performance mapping tool. It also provide a unique tool to evaluate mappability."
+      homepage: "https://paoloribeca.science/gem"
```

Comment:
Is the link valid? It seems that I cannot load the homepage.

Reply:
It is not, all documentation or references to the original tool are gone, but I left this one in because the linter complains if I leave it out